pkg/remote/aws/sns_topic_subscription_supplier.go

```diff
@@ -1,44 +1,63 @@
 package aws
 
 import (
+	"fmt"
+
+	awssdk "github.com/aws/aws-sdk-go/aws"
 	"github.com/aws/aws-sdk-go/service/sns"
-	"github.com/cloudskiff/driftctl/pkg/remote/aws/repository"
-	remoteerror "github.com/cloudskiff/driftctl/pkg/remote/error"
 	"github.com/sirupsen/logrus"
 	"github.com/zclconf/go-cty/cty"
 
+	"github.com/cloudskiff/driftctl/pkg/alerter"
+	"github.com/cloudskiff/driftctl/pkg/remote/aws/repository"
+	remoteerror "github.com/cloudskiff/driftctl/pkg/remote/error"
+
 	"github.com/cloudskiff/driftctl/pkg/remote/deserializer"
 	"github.com/cloudskiff/driftctl/pkg/resource"
 	"github.com/cloudskiff/driftctl/pkg/resource/aws"
 	awsdeserializer "github.com/cloudskiff/driftctl/pkg/resource/aws/deserializer"
 	"github.com/cloudskiff/driftctl/pkg/terraform"
 )
 
+type pendingTopicAlert struct {
+	message string
+}
+
+func (p *pendingTopicAlert) Message() string {
+	return p.message
+}
+
+func (p *pendingTopicAlert) ShouldIgnoreResource() bool {
+	return false
+}
+
 type SNSTopicSubscriptionSupplier struct {
 	reader       terraform.ResourceReader
 	deserializer deserializer.CTYDeserializer
 	client       repository.SNSRepository
 	runner       *terraform.ParallelResourceReader
+	alertr       alerter.AlerterInterface
 }
 
-func NewSNSTopicSubscriptionSupplier(provider *AWSTerraformProvider) *SNSTopicSubscriptionSupplier {
+func NewSNSTopicSubscriptionSupplier(provider *AWSTerraformProvider, alertr alerter.AlerterInterface) *SNSTopicSubscriptionSupplier {
 	return &SNSTopicSubscriptionSupplier{
 		provider,
 		awsdeserializer.NewSNSTopicSubscriptionDeserializer(),
 		repository.NewSNSClient(provider.session),
 		terraform.NewParallelResourceReader(provider.Runner().SubRunner()),
+		alertr,
```

Comment:
typo `alerter`

Reply:
no I named it alertr to prevent package shadowing ^^ I declare 2 alerter...

Reply:
The struct field name is not related to shadowing 😉